model/message.go

```diff
@@ -29,6 +31,27 @@ type Message struct {
 	Headers   http.Header
 	AgeMillis *int
 	QueueName string
+	// TODO: Do we want to add all these keys to Message? Or should they be
+	// parsed as individual strings and added to the span in
+	// processor/otel/consumer.go?
+	RoutingKey string
+	System     string
+	Operation  string
```

Comment:
Can we start out with just the bare minimum, and then expand from there in another PR? The bare minimum being the intersection with what we capture for Elastic APM agents already: queue/topic name.